client/src/utils.ts

```diff
@@ -31,23 +31,31 @@ export const Token = {
 export async function isReadyForConnect(
   appId: string,
   roomId: string,
-  hathoraSdk: HathoraCloud
+  hathoraSdk: HathoraCloud,
 ): Promise<{ lobbyInfo: LobbyV3; connectionInfo: ConnectionDetails }> {
   const MAX_CONNECT_ATTEMPTS = 50;
   const TRY_CONNECT_INTERVAL_MS = 1000;
 
   try {
-    const lobbyInfo = await hathoraSdk.lobbyV3.getLobbyInfoByRoomId(roomId);
+    const lobbyInfo = await hathoraSdk.lobbiesV3.getLobbyInfoByRoomId(roomId);
     if (lobbyInfo.visibility === "local") {
       return new Promise<{ lobbyInfo: LobbyV3; connectionInfo: ConnectionDetails }>((resolve) =>
-        resolve({ lobbyInfo, connectionInfo: LOCAL_CONNECTION_DETAILS })
+        resolve({ lobbyInfo, connectionInfo: LOCAL_CONNECTION_DETAILS }),
       );
     }
 
     for (let i = 0; i < MAX_CONNECT_ATTEMPTS; i++) {
-      const connectionInfoV2 = await hathoraSdk.roomV2.getConnectionInfo(roomId);
+      const connectionInfoV2 = await hathoraSdk.roomsV2.getConnectionInfo(roomId);
       if (connectionInfoV2.exposedPort !== undefined) {
-        return { lobbyInfo, connectionInfo: connectionInfoV2.exposedPort };
+        return {
+          lobbyInfo,
+          connectionInfo: {
+            ...connectionInfoV2.exposedPort,
+            transportType: isRecognized(TransportType, connectionInfoV2.exposedPort.transportType)
```

Comment:
Curious what change makes this needed now?

Reply:
oh the open enums--we now can't assume that the transportType is one of our expected values

Reply:
hmm so now customers using our SDKs need to right a custom function like `isRecognized` to use endpoints with open enums? That seems kinda bad IMO.

Was hoping this would be handled more internal to the SDK. Is that not possible/out of scope?